src/jobs/models/jobsManager.ts

```diff
@@ -349,8 +352,73 @@ export class JobsManager {
       taskId: task.id,
       msg: message,
     });
+
     // eslint-disable-next-line @typescript-eslint/no-magic-numbers
     await this.jobManager.updateJobById(job.id, OperationStatus.COMPLETED, 100, undefined, job.metadata.id);
     job.status = OperationStatus.COMPLETED;
   }
+
+  private async generateSeedJob(
+    job: ICompletedJobs,
+    data: LayerMetadata,
+    previousLayerMetadata: IFindResponseRecord,
+    layerName: string,
+    isSwap = false
+  ): Promise<void> {
+    const cacheName = await this.mapPublisher.getCacheByNameType({ layerName, cacheType: PublishedMapLayerCacheType.REDIS });
```

Comment:
add try catch on this method content, if failed just log it withaout throwing an exception

Reply:
no need on this section - the entire generateSeedJob method wrapped now with try-catch and log warning about the reason seed job creating was failed